src/components/global/Navbar.vue

```diff
@@ -33,16 +57,40 @@ import Tooltip from "@/components/global/Tooltip.vue";
       </v-breadcrumbs>
     </v-toolbar-title>
 
+    <v-btn
+      size="large"
+      :prepend-icon="
+        isFileBrowser || fileBrowserDrawer === true
+          ? 'mdi-source-pull'
+          : 'mdi-folder-open'
+      "
+      variant="tonal"
+      class="session-file-btn text-capitalize font-weight-medium"
+      color="white"
+      @click="
+        isFileBrowser
+          ? router.push('/')
+          : (fileBrowserDrawer = !fileBrowserDrawer)
+      "
+      >{{
+        isFileBrowser
+          ? "View Sessions"
+          : fileBrowserDrawer === true
+            ? "View Session"
```

Comment:
On "top" level, so when not inside a session, this button should say "View Sessions" in the "temporary" file browser. 

So in the Sessions list we have : "All Sessions", "View Session**s**" and "Start New Session"
![image](https://github.com/user-attachments/assets/50aa9446-6e29-48bc-a385-4fb1264cae74)